ibm/service/power/ibm_pi_constants.go

```diff
@@ -371,6 +377,7 @@ const (
 	Attr_UsedIPPercent                               = "used_ip_percent"
 	Attr_UsedMemory                                  = "used_memory"
 	Attr_UserIPAddress                               = "user_ip_address"
+	Attr_UserTags                                    = "user_tags"
```

Comment:
I think this attr is no longer being used (at least in this PR).

Reply:
Removed.